guides/common/modules/proc_configuring-custom-alternate-content-sources.adoc

```diff
@@ -1,5 +1,11 @@
-[id="Configuring_Alternate_Content_Sources_{context}"]
-= Configuring Alternate Content Sources
+[id="Configuring_Custom_Alternate_Content_Sources_{context}"]
+= Configuring Custom Alternate Content Sources
+
+.Prerequisites
+* If the repository requires SSL authentication, import the SSL certificate and key to the {Project}.
+For more information, see {ContentManagementDocURL}Importing_Custom_SSL_Certificates_content-management[Importing Custom SSL Certificates] in _{ContentManagementDocTitle}_. 
+* Note that the alternate content source paths consist of a base URL appended with the subpaths that you provide.
+For example, if your base URL is `https://server.example.com` and your subpaths are `rhel7/` and `rhel8/`, then both `https://server.example.com/rhel7/` and `https://server.example.com/rhel8/` will be searched.
```

Comment:
```suggestion
For example, if your base URL is `https://{foreman-example-com}` and your subpaths are `rhel7/` and `rhel8/`, then both `https://{foreman-example-com}/rhel7/` and `https://{foreman-example-com}/rhel8/` will be searched.
```

Only apply this if you mean "Foreman/Satellite server" with "server".

Reply:
The server in this case will be something external to the Foreman.

Reply:
Then please ignore this comment!